packages/edit-post/src/components/preview-options/index.js

```diff
@@ -22,6 +22,7 @@ import { external, check } from '@wordpress/icons';
 
 const downArrow = (
 	<SVG
+		className="editor-post-preview__button-icon"
```

Comment:
This doesn't seem great

Reply:
Teach me your ways. Whats wrong with it? We could totally do it without the className, but let me know why its bad.

Reply:
I mean this should be something that gets applied by default by using the right ingredients, without having to manually update the SVG.